docs/content/setup/kubectl-plugin.md

```diff
@@ -21,6 +21,9 @@ $ kubectl krew install kcp-dev/kcp
 $ kubectl krew install kcp-dev/ws
 $ kubectl krew install kcp-dev/create-workspace
 ```
+!!! Note
+    When Installed directly from the repo we use `kubectl create workspace`
+    instead of `kubectl create-workspace`
```

Comment:
```suggestion
    When installed directly from the git repository `kubectl create workspace` should be used
    instead of `kubectl create-workspace`.
```